assembl/static2/js/app/app.jsx

```diff
@@ -31,7 +31,7 @@ class App extends React.Component {
   render() {
     const { debateData, debateLoading, debateError } = this.props.debate;
     return (
-      <div className="app">
+      <div className={`${this.props.isHarvesting ? 'app harvesting-mode-on' : 'app'}`}>
```

Comment:
Can leverage `classnames` module here for code synchrony.

Reply:
Good to know, thanks!